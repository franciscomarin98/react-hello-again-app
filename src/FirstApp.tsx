const FirstApp = ({message = "Default message"}) => {

    const greeting = 'Hello World';

    return (
        <>
            <h1>{greeting}</h1>
            <p>{message}</p>
        </>
    );
}

export default FirstApp;