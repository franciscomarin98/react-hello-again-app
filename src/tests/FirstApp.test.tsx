import FirstApp from '../FirstApp'
import {shallow} from 'enzyme';

describe('Test in <FirstApp />', () => {
    /*    test('should show the title message: Hello World', () => {
            const msg = 'Hello World';
            const wrapper = render(<FirstApp title={msg}/>);
            expect(wrapper.getByText(msg)).toBeInTheDocument();
        })*/

    test('should show the component successfully', () => {
        const msg = 'Hello World';
        const wrapper = shallow(<FirstApp title={msg}/>);
        expect(wrapper).toMatchSnapshot();
    })
})