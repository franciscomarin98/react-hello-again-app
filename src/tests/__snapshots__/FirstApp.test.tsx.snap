// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test in <FirstApp /> should show the component successfully 1`] = `
<Fragment>
  <h1>
    Hello World
  </h1>
  <p>
    Default message property
  </p>
</Fragment>
`;
